gtsam/navigation/CombinedImuFactor.h

```diff
@@ -252,7 +252,7 @@ class GTSAM_EXPORT PreintegratedCombinedMeasurements : public PreintegrationType
  *    the correlation between the bias uncertainty and the preintegrated
  *    measurements uncertainty.
  *
- * @ingroup SLAM
+ * @ingroup navigation
```

Comment:
does case matter in the final doc?

Reply:
No but those are labels and I liked the labels to be lower case. Helps differentiate between the actual titles in the docs.